core/ecschema-metadata/src/test/Metadata/Class.test.ts

```diff
@@ -31,6 +31,82 @@ describe("ECClass", () => {
       schema = new Schema(new SchemaContext(), "TestSchema", "ts", 1, 0, 0);
     });
 
+    it("measures caching performance for getProperties", async () => {
```

Comment:
We should not mix performance tests into regular tests. Move this to a better place, like the tests in full-stack-tests/backend/src/perftest/CloseIModalPerformance.test.ts
I see there is a test helper in core/ecschema-metadata/src/test/TestUtils/PerfTestHelper.ts but I cannot find any callers.
We don't seem to have a common structure for performance tests, but let's at least replicate what some existing tests to, like the backend perftest, or core/geometry/src/test/misc/performance.test.ts which puts them into a separate file